index.html

```diff
@@ -140,11 +115,15 @@ <h2 data-key="repo-id"><a href="https://www.github.com/heremaps/{repo-id}" targe
 			$.getJSON('https://api.github.com/orgs/heremaps/repos')
 			.done(function (repo_data) {
 				repo_data.forEach(function (data, index) {
+					/* The GitHub API will return more repos than the ones we expose here.
+						 Target only existing repo sections.*/
 					var $target = $('#repo-info-' + data.name);
 					if ($target.length > 0) {
-						$target.find('[data-role="repo-lastUpdate"]').text(moment(data.updated_at).format('ll'));
-						$target.find('[data-role="fork-count"]').text(data.forks_count);
-						$target.find('[data-role="star-count"]').text(data.stargazers_count);
+						$target.find('[data-key="repo-type"]').text(data.language);
```

Comment:
For the most part we are in agreement:

The code shall inspect the data structure. Should there be a value under the .type property of a record, it should be immediately used. Otherwise use the one the API returns. If that is null, then display an empty string.

I'm confused on the part that shall revert to the _JS_ value **after** the API has returned null.
Let me commit a prototype and let's discuss on that.